### PR TITLE
Query by child value

### PR DESCRIPTION
**Pull request by LuisRodriguezLD**
New implementation, works great on iOS. Sadly can't test on Android but should work

### CODE REVIEW

**Comment by poul-kg** [comment]
@Dhvl-Golakiya please review and test this on both iOS and Android and merge if everything works fine.

**Comment by LuisRodriguezLD** [comment]
FYI, I was _finally_ able to test on Android and it works great

File: src/Firebase.Database/Database.uno [hunk 1]
```diff
@@ -656,6 +656,14 @@ namespace Firebase.Database
         }
     }
 
+    internal class ReadByQueryEqualToValue : Promise<string>
```
Comment (Dhvl-Golakiya) [inline src/Firebase.Database/Database.uno]: You have to declare this class as `extern(!mobile)` We are getting an error in Mobile build with a duplicate declaration.

File: src/Firebase.Database/Database.uno [hunk 2]
```diff
@@ -732,4 +772,50 @@ namespace Firebase.Database
...
+        "com.google.firebase.database.ValueEventListener",
+        "org.json.JSONObject",
+        "org.json.JSONArray",
+        "java.util.Map",
+        "java.util.List")]
+    extern(Android)
+    internal class ReadByQueryEqualToValue : Promise<string>
+    {
+        [Foreign(Language.Java)]
+        public ReadByQueryEqualToValue(string path)
```
Comment (Dhvl-Golakiya) [inline src/Firebase.Database/Database.uno]: Need three parameters `path`, '`key` and `val` for this function.

**Pull request merged**